code/modules/sanity/sanity_mob.dm

```diff
@@ -327,6 +336,13 @@
 				add_rest(taste_tag, 4 * 1/E.taste_tag.len)  //just so it got somme effect of things with small multipliers
 			else
 				add_rest(taste_tag, 4 * multiplier/E.taste_tag.len)
+	if(owner.stats.getPerk(PERK_ARTIST))
```

Comment:
You may want to run this check before `if(resting && E.taste_tag.len)` and add returns at the end.

Reply:
resolved in discord (x3)